packages/commonwealth/server/models/template.ts

```diff
@@ -40,6 +49,10 @@ export default (
       foreignKey: 'abi_id',
       targetKey: 'id',
     });
+    models.Template.belongsTo(models.Chain, {
```

Comment:
Need to be clear about whether we are declaring an association here (in which case we need to use `references` in the migration when creating the column), or not (in which case we remove this line).

Reply:
Added to migration